Update planning_doc.Rmd (#13)
esciencecenter-digital-skills/template-docs-coordination: planning_doc.Rmd

* Participants Spreadsheet 
* Zoom Link 

### Training team:  `r params$training_team`

### Training team general roles:  

> When not teaching, an training team member can do the following things:

Reviewer: ```suggestion
When not teaching, a training team member can do the following things:
```

> * Add more information or helps the other instructor to clarify some points.

Reviewer: ```suggestion
* Add more information or help the other instructor to clarify some points.
```

> * Checks the room for people that are have questions or having difficulties following and can bring this to the attention of the teaching instructor to suggest to spend more time on a certain topic (only if necessary).

Reviewer: ```suggestion
* Check the room for people that have questions or are having difficulties following and can bring this to the attention of the teaching instructor to suggest spending more time on a certain topic (only if necessary).
```

* Can tell teaching Instructor when to slow down or speed up.

> * Walks around to help attendees with small questions.

Reviewer: ```suggestion
* Be on the lookout for help post-its. Walk around to help attendees with small questions.
```

### Lead Instructor:  `r params$lead_instructor`
#### [Lead Instructor is an Instructor who is also responsible for some of the preparatory tasks as well as hosting most of the workshop. They also have the responsibilities of an Instructor (see below).]  
### Lead instrucgtor specific roles:  

* The lead instructor is the responsible person for making sure the workshop is well organized and all content is accessible.
* Prepares the workshop website, collaborative documents, and exercises document.
* Prepares general introduction (see slides for NLeSCs intro) + icebreaker.
* Makes sure to gather feedback from the participants at least every 4 hours (carpentries guidelines) for example via asking for tip/tops.
* Prepares planning, calibration and debriefing meetings.

> 

Reviewer: ```suggestion
* Ensures that everyone on the team knows their roles and responsibilities.
```
